plugin/vim-arsync.vim

```diff
@@ -74,19 +77,19 @@ function! ARsync(direction)
         endif
         if l:conf_dict['remote_or_local'] == 'remote'
             if a:direction == 'down'
-                let l:cmd = [ 'rsync', '-vazre', 'ssh -p '.l:conf_dict['remote_port'], l:user_passwd . l:conf_dict['remote_host'] . ':' . l:conf_dict['remote_path'] . '/', l:conf_dict['local_path'] . '/']
+                let l:cmd = [ 'rsync', '-vazre', l:conf_dict['rsync_flags'], '-e', 'ssh -p '.l:conf_dict['remote_port'], l:user_passwd . l:conf_dict['remote_host'] . ':' . l:conf_dict['remote_path'] . '/', l:conf_dict['local_path'] . '/']
```

Comment:
Not sure what the extra `-e` does here. This option is already specified.

Reply:
Also not sure if it's a good idea to add rsync_flags option here as I already made that field fully customizable in #22